### PR TITLE
Crowdsource updates

### PR DESCRIPTION
**Pull request by jmhauck**

### CODE REVIEW

File: src/components/edit-card/edit-card.tsx [hunk 1]
```diff
@@ -299,15 +287,28 @@ export class EditRecordModal {
...
+
       this._editHandle = this.reactiveUtils.when(
         () => this._editor.viewModel.state === "ready",
         () => {
-          if (this.graphicIndex > -1 && this.graphics.length > 0 && this.open) {
+          if (this._shouldClose) {
+            this.closeEdit.emit();
+            this._shouldClose = false;
+          }
+          if (this.graphicIndex > -1 && this.graphics.length > 0 && this.open && !this._shouldClose) {
```
Comment (MikeTschudi) [inline src/components/edit-card/edit-card.tsx:309]: Is there ever a case where _shouldClose is false without that being set by the previous if clause? The two ifs are coupled in a fragile way if not.

**Review by MikeTschudi — APPROVED** [review]

File: src/components/edit-card/edit-card.tsx (hunk 1 above)
Comment (jmhauck) [reply src/components/edit-card/edit-card.tsx:309]: @MikeTschudi  By default its false and then if they are adding or editing attachments its false. We are working with the JS API to also see if we can get a state change for when adding or editing related records so we could also know if we should close the widget or go back to the main edit panel within the flow.

File: src/components/crowdsource-manager/crowdsource-manager.tsx [hunk 2]
```diff
@@ -153,8 +153,8 @@ export class CrowdsourceManager {
                 {this._translations.layout}
               </div>
               {this._getAction("grid-background", ELayoutMode.GRID, this._translations.grid)}
-              {this._getAction("horizontal-background", ELayoutMode.VERTICAL, this._translations.horizontal)}
-              {this._getAction("vertical-background", ELayoutMode.HORIZONTAL, this._translations.vertical)}
+              {this._getAction("horizontal-background", ELayoutMode.VERTICAL, this._translations.vertical)}
+              {this._getAction("vertical-background", ELayoutMode.HORIZONTAL, this._translations.horizontal)}
             </div>
```
[the remark this answers is not in the document]
Comment (jmhauck) [reply src/components/crowdsource-manager/crowdsource-manager.tsx:158]: @MikeTschudi yes...we can. The actual handling of these layouts will be done within the Instant App itself and this code won't actually be used for what we release. I just made it so we have a place to test how things work together from the demos in an easier way but clarity even for the demos is a good idea.

File: src/components/edit-card/edit-card.css [hunk 3]
```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-:host {
+ :host {
```
[the remark this answers is not in the document]
Comment (jmhauck) [reply src/components/edit-card/edit-card.css:17]: will update this and the other in a subsequent PR

**Pull request merged**